src/bridge.js

```diff
@@ -72,15 +79,27 @@ try {
 } catch (error) {
   // epBridge was not found. In such case, we assume that the application is iframed within
   // WebPlayer and communication has to proceed via posting and receiving messages between windows.
-  // TODO(michal): generalize hardcoded player.enplug.loc URL.
-  console.info('Initializing Web Development Player.');
+  console.info(`[Player SDK: ${VERSION}] Initializing Web Development Player.`);
+  const destinationMatch = window.location.href.match(/(https\:\/\/[a-z]*\.[a-z]*\.[a-z]{3})/);
```

Comment:
This won't match "https://player.enplug.in" (last group requires 3 characters)

Reply:
Done